scripts/ipintutil

```diff
@@ -150,7 +150,10 @@ def get_ip_intfs_in_namespace(af, namespace, display):
         ip_intf_attr = []
         if namespace != constants.DEFAULT_NAMESPACE and skip_ip_intf_display(iface, display):
             continue
-        ipaddresses = multi_asic_util.multi_asic_get_ip_intf_addr_from_ns(namespace, iface)
+        try:
+            ipaddresses = multi_asic_util.multi_asic_get_ip_intf_addr_from_ns(namespace, iface)
+        except ValueError:
+            continue
```

Comment:
Is it better to print error message and exit? If you continue, `ipaddresses` is not initialized and used.

Reply:
I don't think its correct. This will avoid other interfaces detail to not be shown. This is a very rare scenario during config reload.
Here is the scenario - While getting the interfaces there are 4 ports (docker0, eth0, lo0 and Loopback0). But while it enters the for loop in the parallel config reload flow Loopback0 gets deleted. However other interfaces exist and those information should be displayed. We can skip Loopback0 as we get valueerror meaning the interface does not exist now

Reply:
Sorry, there is no uninitialized variable in use.